types/time.go

```diff
@@ -1368,13 +1368,13 @@ func checkDatetimeType(t MysqlTime, allowZeroInDate bool) error {
 
 	hour, minute, second := t.Hour(), t.Minute(), t.Second()
 	if hour < 0 || hour >= 24 {
-		return errors.Trace(ErrInvalidTimeFormat)
+		return errors.Trace(ErrInvalidTimeFormat.GenByArgs(hour))
 	}
 	if minute < 0 || minute >= 60 {
-		return errors.Trace(ErrInvalidTimeFormat)
+		return errors.Trace(ErrInvalidTimeFormat.GenByArgs(minute))
```

Comment:
Do we need to add the minute field in the log like `fmt.Sprintf("minute: %v", minute)` or use `t` as the arg ?

Reply:
The error stack log would have enough information the trace to this line, so the minute value is enough